src/interfaces/simulation.ts

```diff
@@ -482,6 +484,10 @@ export class SimulationInterface<T extends ChainId> extends ServiceInterface<T>
     return result;
   }
 
+  private shouldUsePartnerService(vault: string): boolean {
+    return !!this.yearn.services.partner && this.yearn.services.partner.isAllowed(vault);
```

Comment:
We could probably simplify to just be

```suggestion
    return this.yearn.services.partner?.isAllowed(vault);
```

Reply:
Fixed on 03bf09134a3c47a2521dad867c47195f8c52bfca

Reply:
Nice, thank you for leaving the commit Fibo!!! Looks great 👍